### PR TITLE
add quarkus news blog + john clingan link list

### PR DESCRIPTION
**Pull request by maxandersen**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

File: _posts/2019-10-01-quarkus-news-1.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: 'Quarkus News #1'
```
Comment (emmanuelbernard) [inline _posts/2019-10-01-quarkus-news-1.adoc]: @maxandersen do you think people will get what it is about? It's not about what's new in Quarkus. More about articles people write.

Comment (maxandersen) [reply _posts/2019-10-01-quarkus-news-1.adoc]: I grok your concern - I went for this as it was short and sweet...others I considered was "weekly" (i.e. "Docker Weekly") but didn't want to commit to that kind of schedule (yet :)

Jboss has has "weekly editorial" ....I guess "Quarkus Editorial" would work - just sounds so...dry.

any ideas ?

Comment (maxandersen) [reply _posts/2019-10-01-quarkus-news-1.adoc]: Quarkus Gazette ?  Qazette ?

Comment (maxandersen) [reply _posts/2019-10-01-quarkus-news-1.adoc]: QNews ? (since QTips are used already ?)

**Comment by maxandersen** [comment]
"Quarkus Newsletter"

File: _posts/2019-10-01-quarkus-news-1.adoc (hunk 1 above)
Comment (jclingan) [reply _posts/2019-10-01-quarkus-news-1.adoc]: FWIW, I've always considered myself more of a hashmap than a linked list.

Comment (jclingan) [reply _posts/2019-10-01-quarkus-news-1.adoc]: Per above, perhaps replace Quarkus with Q:
Quarkus Report
Quarkus News
Quarkus Insider
Quarkus Times

QZone
InfoQ. Wait, nevermind :-)

Comment (maxandersen) [reply _posts/2019-10-01-quarkus-news-1.adoc]: this was published earlier with "Quarkus Newsletter #1" :)

**Comment by emmanuelbernard** [comment]
I like the Quarkus Gazette idea. Maybe for round 2

**Pull request merged**